src/Compilers/CSharp/Portable/Binder/Imports.cs

```diff
@@ -286,6 +291,12 @@ public static Imports FromGlobalUsings(CSharpCompilation compilation)
                 var imported = usingsBinder.BindNamespaceOrTypeSymbol(qualifiedName, diagnostics).NamespaceOrTypeSymbol;
                 if (uniqueUsings.Add(imported))
                 {
+                    if (imported is NamespaceSymbol ns)
+                    {
+                        Debug.Assert(!ns.IsGlobalNamespace);
+                        compilation.AddAssembliesUsedByNamespaceReference(ns);
```

Comment:
>compilation [](start = 24, length = 11)

Is this line tested?

Reply:
> Is this line tested?

Yes.

---
In reply to: [346974882](https://github.com/dotnet/roslyn/pull/39807#discussion_r346974882) [](ancestors = 346974882)